interns-react-weekly-assignments/src/components/Home/Shreyansh/Week_4/components/Feature2/Feature2.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import { Feature2Wrapper } from "./Feature2Style";
```

Comment:
I recommend to have proper name of file and folder.
instead of feature2, it should be calculator